@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -198,6 +190,25 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
             }
             virtualFocus.moveFocusUp();
           }
+        } else if (e.key === "Home") {
+          e.preventDefault();
+          toggleIsListOpen(false);
+          virtualFocus.moveFocusToTop();
+        } else if (e.key === "End") {
+          e.preventDefault();
+          toggleIsListOpen(true);
+          virtualFocus.moveFocusToBottom();
+        } else if (e.key === "PageUp") {
+          e.preventDefault();
+          virtualFocus.moveFocusUpBy(6);
+          return false;
```

Comment:
What does `return false` do here and in ` else if (e.key === "PageDown")` ?

Reply:
Nothing. 🙈 It was a step in trying to prevent page scrolling, but had no effect. Removing it now.